src/components/OurTeam/OurTeam.jsx

```diff
@@ -0,0 +1,94 @@
+import styles from './OurTeam.module.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTwitter, faInstagram, faFacebook, faLinkedin } from '@fortawesome/free-brands-svg-icons';
+
+function OurTeam({isDarkMode}) {
```

Comment:
I don't think you are using the isDarkMode prop here though, so you can remove it